utm.yaml

```diff
@@ -1317,6 +1271,213 @@ components:
         status:
           $ref: '#/components/schemas/UssAvailabilityStatus'
 
+    RequestResponse:
+      type: object
+      description: Details of a request/response data exchange.
+      required:
+      - url
+      - method
+      - request_time
+      - request_body
+      - response_time
+      - response_body
+      - response_code
+      properties:
+        url:
+          type: string
+          description: Full URL of request.
+        method:
+          type: string
+          description: HTTP verb used by requestor (e.g., "PUT," "GET," etc.).
+        headers:
+          description: Set of headers associated with request or response.
+          type: array
+          items:
+            type: string
+          default: []
+        request_time:
+          description: The time at which the request was sent/received.
+          allOf:
+          - $ref: '#/components/schemas/Time'
+        request_body:
+          description: Base64-encoded body content sent/received as a request.
+          type: string
+        response_time:
+          description: The time at which the response was sent/received.
+          allOf:
+          - $ref: '#/components/schemas/Time'
+        response_body:
+          type: string
+          description: Base64-encoded body content sent/received in response to request.
+        response_code:
+          type: integer
+          format: int32
+          description: HTTP response code sent/received in response to request.
+
+    ErrorReport:
+      type: object
+      description: A report informing a server of a communication problem.
+      required:
+      - exchange
+      - problem
+      properties:
+        report_id:
+          type: string
+          maxLength: 128
+          description: ID assigned by the server receiving the report.  Not populated
+            when submitting a report.
+        exchange:
+          allOf:
+          - $ref: '#/components/schemas/RequestResponse'
+          description: The request (by this USS) and response associated with the error.
+        problem:
+          type: string
+          description: Human-readable description of the problem with the response.
+
+    ExchangeRecord:
+      type: object
+      description: |-
+        A record of data exchange (USS - USS or USS - DSS).
+      required:
+      - request_source
+      - exchange
+      properties:
+        request_source:
+          type: string
+          description: |-
+            A coded value that indicates the requesting USS: 'LOCAL' (this USS) or 'REMOTE' (peer USS)
+          enum:
+          - LOCAL
```

Comment:
To match style with the rest of the document, enum strings should be in normal caps (e.g., "Local", "Remote")

Reply:
Done, but there are many more 'enum' values that are all-caps in the YAML. Should we convert these all to just "capitalized?"